### PR TITLE
Use caml_alloc_custom_mem when available 

### PR DESCRIPTION
**Pull request by hhugo**

### CODE REVIEW

**Comment by hhugo** [comment]
cc @stedolan

File: src/win32/draw.c [hunk 1]
```diff
@@ -432,7 +433,7 @@ struct image {
 #define Height(i) (((struct image *)Data_custom_val(i))->h)
 #define Data(i) (((struct image *)Data_custom_val(i))->data)
 #define Mask(i) (((struct image *)Data_custom_val(i))->mask)
-#define Max_image_mem 500000
+#define Max_image_mem 2000000
```
Comment (jeremiedimino) [inline src/win32/draw.c:436]: Why this change?

Comment (hhugo) [reply src/win32/draw.c:436]: Reason is in the commit msg: "Synchronize Max_image_mem between unix and win32 backends"

Comment (jeremiedimino) [reply src/win32/draw.c:436]: Got it

**Comment by jeremiedimino** [comment]
Are you happy to do a new minor release for this?
